mobilenet/README.md

```diff
@@ -88,19 +88,13 @@ mobilenet.load(
 
 Args:
 - **version:** The MobileNet version number. Use 1 for [MobileNetV1](https://github.com/tensorflow/models/blob/master/research/slim/nets/mobilenet_v1.md), and 2 for [MobileNetV2](https://github.com/tensorflow/models/tree/master/research/slim/nets/mobilenet). Defaults to 1.
-- **alpha:** Controls the width of the network, trading accuracy for performance. A smaller alpha decreases accuracy and increases performance. Defaults to 1.0.
+- **alpha:** Controls the width of the network, trading accuracy for performance. A smaller alpha decreases accuracy and increases performance. Defaults to 1.0 and 0.25 is only for V1.
```

Comment:
how about two sentences here:

`0.25 is only available for V1. Defaults to 1.0.`